style.css

```diff
@@ -0,0 +1,252 @@
+/* ------------General styles--------- */
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f4f5f6;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+/* ------------Header--------- */
+.red {
+  background-color: red;
+}
+
+.float-c {
+  transform: translate(-50%, 0);
+  position: absolute;
+  z-index: 3;
+  top: 0;
+  left: 50%;
+  padding: 10px 0;
+}
+
+.container-inline {
+  float: right;
+}
+
+.container-fluid {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.container-fluid .container {
+  max-width: 1310px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+@media screen and (min-width: 992px) {
+  .page-footer::before {
+    left: -30px;
+    right: -30px;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .flex-sm {
+    display: flex;
+  }
+}
+
+.jc-sb {
+  justify-content: space-between;
+}
+
+.ai-fe {
+  align-items: flex-end;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    font-size: 15px;
+    line-height: 20px;
+  }
+}
+
+.page-footer .copyright {
+  font-size: 12px;
+  line-height: 12px;
+  text-transform: uppercase;
+  color: rgb(102, 102, 102);
+  margin-bottom: 12px;
+}
+
+.page-footer .social {
+  color: rgb(102, 102, 102);
+  margin-bottom: 12px;
+}
+
+.flex-xs {
+  display: -webkit-flex;
+  display: flex;
+}
+
+li {
+  display: list-item;
+}
+
+.page-footer .brand svg {
+  display: block;
+  width: 236px;
+  height: 36px;
+  margin-right: 15px;
+  margin-bottom: 12px;
+  fill: rgb(247, 34, 16);
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
```

Comment:
- [ ] This can be done with the bootstrap class `text-decoration-none` and `text-dark`